engine/src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -61,6 +62,7 @@ case class WorkflowDescriptor(id: WorkflowId,
     log.warn(s"$CallLogsDirOptionKey expected to be of type String", e)
     throw e
   }
+  val workflowFailureMode: WorkflowFailureMode = workflowOptions.get(WorkflowFailureModeKey) flatMap {WorkflowFailureMode.tryParse} getOrElse configWorkflowFailureMode
```

Comment:
It looks as if `WorkflowFailureMode` is specified incorrectly this would silently ignore that and just go with what's in the config?